### PR TITLE
Added example for the identities of a multikey setup

### PR DESCRIPTION
**Pull request by iulianpascalau**
#### Description of the pull request (what is new / what has changed)
- added example for the identities of a multikey setup

#### Did you test the changes locally ?
- [X] yes
- [ ] no

#### Which category (categories) does this pull request belong to?
- [ ] document new feature
- [ ] update documentation that is not relevant anymore
- [X] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/validators/key-management/multikey-nodes.md [hunk 1]
```diff
@@ -163,10 +163,151 @@ At the first sight, this can be seen as a security degradation in terms of means
...
+   # Possible values: "round", "nonce", or "epoch"
+   CutoffTrigger = "round"
+
+   # The minimum value of the cutoff. For example, if CutoffType is set to "round", and Value to 20, then the node will stop processing at round 20+
+   Value = 0
+
+# NamedIdentity represents an identity that runs nodes on the multikey
+# There can be multiple identities set on the same node, each one of them having different bls keys, just by duplicating the NamedIdentity
+[[NamedIdentity]]
+   # Identity represents the keybase/GitHub identity for the current NamedIdentity
```
Comment (raduchis) [inline docs/validators/key-management/multikey-nodes.md:280]: also here is keybase mentioned

**Review by sstanculeanu — APPROVED** [review]

File: docs/validators/key-management/multikey-nodes.md [hunk 2]
```diff
@@ -163,10 +163,151 @@ At the first sight, this can be seen as a security degradation in terms of means
 
 Regarding point 3, each managed BLS key will create a virtual p2p identity that no node from the network can connect to since it does not advertise the connection info but is only used to sign p2p messages.
 Associated with a separate named identity, the system will make that BLS key virtually unreachable, and its origin hidden from the multikey nodes. Therefore, the node operators will need to apply the following changes on the `prefs.toml` file:
-* in the `[Preference]` section, the 2 options called `NodeDisplayName` and `Identity` will be changed to something relevant for the nodes that are running the multikey group;
+* in the `[Preference]` section, the 2 options called `NodeDisplayName` and `Identity` will be changed to something different used in the BLS' definitions to prevent easy matching. Generic names like `gateway` or `observer` are suitable for this section.
```
Comment (raduchis) [inline docs/validators/key-management/multikey-nodes.md:166]: Generic names like `gateway` or `observer` are **NOT** suitable for this section?

File: docs/validators/key-management/multikey-nodes.md [hunk 3]
```diff
@@ -163,10 +163,151 @@ At the first sight, this can be seen as a security degradation in terms of means
...
+   # DestinationShardAsObserver represents the desired shard when running as observer
+   # value will be given as string. For example: "0", "1", "15", "metachain"
+   # if "disabled" is provided then the node will start in the corresponding shard for its public key or 0 otherwise
+   DestinationShardAsObserver = "0"
+
+   # NodeDisplayName represents the friendly name a user can pick for his node in the status monitor when the node does not run in multikey mode
+   # In multikey mode, all bls keys not mentioned in NamedIdentity section will use this one as default
+   NodeDisplayName = "observer"
+
+   # Identity represents the keybase/GitHub identity when the node does not run in multikey mode
```
Comment (raduchis) [inline docs/validators/key-management/multikey-nodes.md:217]: should we still mention keybase?

File: docs/validators/key-management/multikey-nodes.md (hunk 2 above)
Comment (raduchis) [inline docs/validators/key-management/multikey-nodes.md:166]: called `NodeDisplayName` and `Identity` _will_ -> _should_ be changed to something different

Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:166]: are suitable. As they are too generic to show you some links with the keys managed.

File: docs/validators/key-management/multikey-nodes.md (hunk 1 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:280]: removed, thanks

File: docs/validators/key-management/multikey-nodes.md (hunk 2 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:166]: added also the possibility of having random strings. Used random string in the example

File: docs/validators/key-management/multikey-nodes.md (hunk 3 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:217]: removed, thanks

File: docs/validators/key-management/multikey-nodes.md (hunk 2 above)
Comment (iulianpascalau) [reply docs/validators/key-management/multikey-nodes.md:166]: changed

**Review by raduchis — APPROVED** [review]

**Review by sstanculeanu — APPROVED** [review]

**Pull request merged**